checkov/common/util/type_forcers.py

```diff
@@ -0,0 +1,10 @@
+def force_list (var):
+    if not isinstance(var, list):
+        var = [var]
+    return var
+
+
+def force_int (var):
+    if not isinstance(var, int):
+        var = int(var)
```

Comment:
@arielkru I would suggest having a try-catch here, and in case of an exception return `None`